exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -500,6 +500,16 @@ class CesiumPolygonRasterOverlayPrim "CesiumPolygonRasterOverlayPrim" (
         displayName = "Cartographic Polygon Binding"
         doc = "Specifies which Cartographic Polygons to use in the raster overlay"
     )
+
+    bool cesium:invertSelection = false (
```

Comment:
Any new USD fields need to be accounted for in `UsdNotificationHandler`. In this case, should be a simple change in `processCesiumPolygonRasterOverlayChanged`.